02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,86 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
   <link rel="stylesheet" href="css/style.css">
+  <title>Zac Collier Music</title>
+
+<!--  HEADER -->
```

Comment:
It'll still look okay... because HTML can be callous... but it will create some parsing errors when a machine is reading your page or if your user is using a screen reader.